201-ospatching-extension-on-ubuntu/azuredeploy.json

```diff
@@ -119,7 +151,7 @@
     "publicIPAddressName": "myPublicIP",
```

Comment:
Please use a name that follows this scheme for these variables - templateScenarioResourceName